backup/build.gradle.kts

```diff
@@ -27,8 +27,10 @@ plugins {
     alias(libs.plugins.kotlinNativeCoroutines)
 }
 
+version = "0.0.1"
```

Comment:
Sugestion: We could preemptively set the version at 0.0.2 already since 0.0.1 is already deployed

Reply:
I've done it in the other branch, where the fixes and user verification API for Web are being developed :)